app/models/form526_submission.rb

```diff
@@ -157,12 +157,29 @@ class Form526Submission < ApplicationRecord
   belongs_to :user_account, dependent: nil, optional: true
 
   validates(:auth_headers_json, presence: true)
+  enum backup_submitted_claim_status: { accepted: 0, rejected: 1 }
```

Comment:
will this allow `nil`?  I needs to, if not

Reply:
Yes, an instance's `backup_submitted_claim_status` attribute value is not prevented from being `nil`.